### DISCUSSION TITLE
Vlang doesn't install on PCLos LINUX

### DISCUSSION THREAD

**Discussion by BugzRus** [vlang/v#17706] [Installation Issues]
I installed Vlang on Debian Bullseye two weeks ago no problems. Needed to install on PCLos kernel 6.0.12 today  -- using both git and the zip file. Neither was successful.  Attempt with Git and 'make' result was
```
cd ./vc && git clean -xf && git pull --quiet
cd ./thirdparty/tcc && git clean -xf && git pull --quiet
cc  -std=gnu99 -w -o v1.exe ./vc/v.c -lm -lpthread 
./v1.exe -no-parallel -o v2.exe  cmd/v
==================
tcc: error: library 'atomic' not found
...
==================
(Use `v -cg` to print the entire error message)

builder error: 
==================
C error. This should never happen.
```
RESULT with .zip  file ----
```
cd ./vc && git clean -xf && git pull --quiet
cd ./thirdparty/tcc && git clean -xf && git pull --quiet
cc  -std=gnu99 -w -o v1.exe ./vc/v.c -lm -lpthread 
./v1.exe -no-parallel -o v2.exe  cmd/v
vlib/v/token/keywords_matcher_trie.v:73:18: error: cannot use `token.Kind` as `int` in argument 2 to `v.token.KeywordsMatcherTrie.add_word`
   71 |     }
   72 |     for k, v in kw_map {
   73 |         km.add_word(k, v)
      |                        ^
   74 |     }
   75 |     // dump(km.min_len)
make: *** [GNUmakefile:114: all] Error 1
[poco@localhost v]$ name -a
bash: name: command not found
[poco@localhost v]$ uname -a
Linux localhost.localdomain 6.0.12-pclos1 #1 SMP PREEMPT_DYNAMIC Thu Dec 8 21:11:54 CST 2022 x86_64 x86_64 x86_64 GNU/Linux
```
I have GCC compiler installed. I will try another compiler if possible.

Thanks for any assistance.

**Comment by JalonSolov** [comment]
Check the system packages for the dev files for `libatomic`, often just `libatomic-dev`.

Once that is installed, try `make` for V again.

**Reply by BugzRus** [reply]
Thanks for the suggestion. I downloaded and installed a copy of TCC.  I don't know if it included a copy of Libatomic. At any rate it's there.  I did 'make' again and got a similar error message --  but the v compiler works.. I'll settle with that for now, and see if I can find a complete solution. Thanks again.

**Reply by JalonSolov** [reply]
No, tcc doesn't include it.  V installs a version of tcc for you in `thirdparty/tcc`, so you already had one... and likely newer than what you can install from system packages.  Same version, but what V installs for you is 2 years of commits past that version.

**Comment by BugzRus** [comment]
Thank you Jalon. I'll try installing it again without the added tcc, and
see if it works.
